README.md

```diff
@@ -360,6 +366,8 @@ Call Signatures:
   - `plugins`: Array of Rollup, Vite, or WMR plugins to add.
   - `envVars`: Object with string keys and string values for environment variables to pass in as `import.meta.env.*` / `process.env.*`
   - `esbuild`: [`TransformOptions`](https://esbuild.github.io/api/#transform-api) | `false`: Options to pass to esbuild. Set to false to disable esbuild.
+- `user`: User API options object (all properties are optional). They will be applied when calling `user.*` methods.
+  - `targetSize`: (`number | boolean`) Set the minimum target size for `user.click`. Set to `false` to disable target size checks. This option can also be passed to individual `user.click` calls in the 2nd parameter.
```

Comment:
Can we include the default value here?

Reply:
Done https://github.com/cloudfour/pleasantest/pull/248/commits/c962fa3cb102f3a9fb44f20ff2590c35b35bd6bc